prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+
+console.log('languages array', languages);
+
+var firstElement = languages[0];
+
+console.log('the first element of the array is', firstElement);
+
+var thirdElement = languages[2];
+
+console.log('the 3rd element of the array is', thirdElement);
+
+var length = languages.length;
+
+console.log('the length of the array is', length);
+
+var lastElement = languages[6 - 1];
+
```

Comment:
Here you can subtract `1` from your `length` variable.

```js
var lastElement = languages[length - 1];
```

Reply:
Thanks, I fixed it!
Should I submit the code again?

Reply:
Yes please 👍

Reply:
I updated the code. Are you able to see it? Do I need to open a new pull request?